src/momento/retry/default_eligibility_strategy.py

```diff
@@ -28,32 +28,57 @@
 ]
 
 RETRYABLE_REQUEST_TYPES: list[str] = [
```

Comment:
Compare to the [Go SDK version](https://github.com/momentohq/client-sdk-go/blob/9ebbbe63aefbfdfee3c0fcbe574973980838b54b/internal/retry/eligibility_strategy.go#L16).